app/org/sagebionetworks/bridge/models/surveys/NumericalConstraints.java

```diff
@@ -10,7 +10,7 @@
     private Double step;
     
     public NumericalConstraints() {
-        setSupportedHints(EnumSet.of(UIHint.NUMBERFIELD, UIHint.SLIDER));    
+        setSupportedHints(EnumSet.of(UIHint.NUMBERFIELD, UIHint.SLIDER, UIHint.SELECT));
```

Comment:
I would add validation that minValue and maxValue are set for the slider and select uiHints, because without these, I don't know how the client can render them. The step value has a default value of 1 or 1.0, so it remains optional.